### PR TITLE
MultiServer: introduce LocationScouts create_as_hint -> only_new

### PR DESCRIPTION
**Pull request by Berserker66**

### CODE REVIEW

**Comment by Alchav** [comment]
👌👌👌

File: MultiServer.py [hunk 1]
```diff
@@ -1534,7 +1537,7 @@ async def process_client_cmd(ctx: Context, client: Client, args: dict):
 
         elif cmd == 'LocationScouts':
             locs = []
-            create_as_hint = args.get("create_as_hint", False)
+            create_as_hint: int = int(args.get("create_as_hint", 0))
```
Comment (Ijwu) [inline MultiServer.py:1540]: What happens when this fails because clients aren't up to date. Does `int("true")` result in `1`? Is this backwards compat?

Comment (Berserker66) [reply MultiServer.py:1540]: json is already converted, so you would only get True or False, never "true", and int(True) is indeed 1.
```python
>>> int(True)
1
```

Comment (Ijwu) [reply MultiServer.py:1540]: I know `int(True) == 1`, just making sure we end up that way. I wasn't sure of the conversion happened. This is kind of... luckily backwards compatible. Not obvious at all, which probably isn't an issue just kind of an odd thing in my mind.

Comment (dewiniaid) [reply MultiServer.py:1540]: My Zen of Python "Explicit is better than implicit." senses say that if we're intentionally using this behavior in this fashion we should maybe have a comment stating this, but I'm not going to fight for this.

Comment (Berserker66) [reply MultiServer.py:1540]: we are quite explicitly converting to int, are we not?

**Pull request merged**